CMakeLists.txt

```diff
@@ -26,8 +26,8 @@ if (examples)
   add_subdirectory(Examples)
 endif()
 
-find_package (PFUNIT 4.0.1 QUIET)
-if (PFUNIT_FOUND)
+if (BUILD_TESTING)
+  find_package (PFUNIT 4.0.1 REQUIRED)
```

Comment:
Doesn't this change also require  `include(CTest)` to add `BUILD_TESTING` as an option?   And if it does, then the default build will brek when pFUnit is not found.  No?

Reply:
I actually thought the default value was OFF---you're right though that the default is ON. Here, `include(CTest)` is omitted so `BUILD_TESTING` isn't defined and the `if()` evaluates to false. Therefore the default build is okay.

I don't think this change is necessary since gFTL, gFTL-shared, fArgParse, and pFUnit are embeddable; also, since test targets aren't included in the "all" target, I don't see a need for such a switch.